themes/finna2/templates/RecordDriver/DefaultRecord/popover-link-element.phtml

```diff
@@ -0,0 +1,26 @@
+<?php
+$aAttributes = $this->htmlAttributes(['class' => $this->classes, 'href' => '#']);
+$aAttributes->add('class', 'show-info');
+$aAttributes->add('title', $this->title ?: $this->translate('field_info_link_title'));
+$additionalDataElem = !empty($this->additionalData) ? "<span class=\"field-additional-data\">{$this->additionalDataHtml}</span>" : '';
```

Comment:
!empty($this->additionalData) pitäisi varmaan tarkistaa additionalDataHtml tässä?

Reply:
Korjattu. Tästä seurasi rumuutta, jota vielä täytyy viilata.